backend/spec/services/api_sirene_spec.rb

```diff
@@ -87,4 +87,63 @@
       expect(subject).to eq(nil)
     end
   end
+
+  describe "cache management" do
+    let(:siret) { "21920023500014" }
+    let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store, expires_in: 514532) }
```

Comment:
Can you explain in a comment why do you choose this very specific value for explires_in ?

Reply:
I've added a comment explaining this time.